server/region_syncer/client.go

```diff
@@ -56,6 +56,7 @@ func (s *RegionSyncer) establish(addr string) (ClientStream, error) {
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
+	defer cc.Close()
```

Comment:
It closed, can the returned `ClientStream` be used?

Reply:
cc is an instance of grpc.ClientConn, not ClientStream

Reply:
there is something wrong. i have fixed it